developers/uploader/uploader-js.md

```diff
@@ -0,0 +1,51 @@
+# Uploader.js
+
+<a href="https://github.com/oceanprotocol/uploader.js" target="_blank">Uploader.js</a> is a robust TypeScript library that serves as a vital bridge to interact with the Ocean Uploader API. It simplifies the process of managing file storage uploads, obtaining quotes, and more within the Ocean Protocol ecosystem. This library offers developers a straightforward and efficient way to access the full range of Uploader API endpoints, facilitating seamless integration of decentralized storage capabilities into their applications. 
+
+Whether you're building a decentralized marketplace, a content management system, or any application that involves handling digital assets, Uploader.js provides a powerful toolset to streamline your development process and enhance your users' experience.
+
+### Browser Usage
+
+Ensure that the Signer object (signer in this case) you're passing to the function when you call it from the browser is properly initialized and is compatible with the browser. For instance, if you're using something like MetaMask as your Ethereum provider in the browser, you'd typically use the ethers.Web3Provider to generate a signer.
+
+1. Setting up a Signer: with MetaMask or similar browser wallets, you can set up an ethers signer as follows:
+
+```javascript
+const provider = new Web3Provider(window.ethereum)
+const signer = provider.getSigner()
+```
+
+2. Initialize UploaderClient:
+
+3. HTML Setup: Provide a file input for users to select multiple files.
+
+```html
+<input type="file" multiple id="fileInput" />
+```
+
+4. JavaScript: Get the files from the input and call the upload function.
+
+```javascript
+document.getElementById('fileInput').addEventListener('change', async function () {
+  const files = this.files
+  await uploadBrowser(quoteId, tokenAddress, files)
+})
+```
+
+For additional details, please visit the Uploader.js repository.
+
+### API
+
+The library provides the following methods:
+constructor(baseURL: string): Create a new instance of the UploaderClient.
+- ```getStorageInfo(): Promise<StorageInfo[]>```: Fetch information about supported storage types and payments.
```

Comment:
The information in this paragraph is quite difficult to go through visually. Nothing stands out or is bolted. I suggest to use the expandable component